elements/pfe-sass/src/mixins/_mixins.scss

```diff
@@ -0,0 +1,154 @@
+// $boxSize = any box-sizing property, default is border-box
+@mixin pfe-box-sizing($boxSize: border-box) {
+  *, *::before, *::after {
+    box-sizing: $boxSize;
+  }
+}
+
+@mixin pfe-clearfix {
+  &::after {
+    clear: both;
+    content: "";
+    display: table;
+  }
+}
+
+@mixin pfe-reset-list() {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+@mixin pfe-reset-box() {
+  margin: 0;
+  padding: 0;
+}
+
+@mixin pfe-sr-only() {
+  position: absolute;
+  overflow: hidden;
+  clip: rect(0,0,0,0);
+  height: 1px;
+  width: 1px;
+  margin: -1px;
+  padding: 0;
+  border: 0;
+}
+
+@mixin pfe-transition($properties...) {
+  $return: null;
+  @each $property in $properties {
+    @if length($property) == 1 {
+      $return: append($return, #{$property} 250ms $pfe-global--animation-timing, comma);
+    }
+    @else if length($property) == 2 {
+      $return: append($return, nth($property, 1) nth($property, 2), comma);
+    }
+  }
+  transition: $return;
+}
+
+// Make a CSS arrow
+@mixin pfe-arrow($type, $color, $width: 8px, $height: 5px, $pseudo: false) {
+  $direction: (
+    up: 180deg,
+    down: 0deg,
+    right: -90deg,
+    left: 90deg,
+  );
+  border-style: #{pfe-var(surface--border-style)};
+  border-width: $width $height 0;
+  border-color: transparent;
+  border-top-color: $color;
+  transform: rotate(map-get($direction, $type));
+  @if $pseudo {
+    display: inline-block;
+    content: "";
+    @content;
+  }
+}
+
+@mixin pfe-arrow-color($color) {
+  border-top-color: $color;
+}
+
+@mixin pfe-print-media {
+  @media print {
+    @content;
+  }
+}
+
+@mixin pfe-print-background {
+  -webkit-print-color-adjust: exact;
+  color-adjust: exact;
+}
+// scss-lint:disable ImportantRule
+@mixin pfe-no-print-background {
+  @include pfe-print-media {
+    background-color: white !important;
+    background-image: none !important;
+    box-shadow: none !important;
+  }
+}
+// scss-lint:enable ImportantRule
+
+@mixin pfe-pipe( $position: right, $padding: 0, $margin: 0, $color: $border-color) {
+  border: #{pfe-var(surface--border-width)} #{pfe-var(surface--border-style)} transparent;
+  border-#{$position}-color: $color;
+  @if $position == right {
+    padding-left: $padding;
+    margin-right: $margin;
+  }
+  @else if $position == left {
+    padding-right: $padding;
+    margin-left: $margin;
+  }
+}
+
+@mixin pfe-retina {
+  @media (-webkit-min-device-pixel-ratio: 2), (min-resolution: 192dpi) {
+    @content;
+  }
+}
+
+// New broadcast function, for use with content components
+//  1st arg:  lookup surface value from theme
+//  2nd arg:  list of broadcast variable  (1 of 6) - optional
+//  3rd arg:  identify the color set, default is surface - optional
```

Comment:
what does "identify the color set" mean? Can we include two examples, one that shows the simple version (1st arg only) and one that shows all 3 arguments?

Reply:
is the usage example showing it as if it were a function, but its a mixin so it should be like this right?
```
.on-dark {
  @include pfe-set-broadcast(darkest);
 }
```

Reply:
Yes that's right.  If you wanted to pull in a different set of colors like the feedback colors, you could do that this way:
```
@each $color in (critical, important, moderate, success, info, default) {
    @include pfe-set-broadcast($color: #{$color}, $broadcast: text, $color-set: feedback);
}
```